### PR TITLE
Add /api/runs/{id} route

### PR DESCRIPTION
**Pull request by lukebjerring**
## Description
Working towards #213 

Adds a route for fetching a specific run by its ID.

## Review Information
See the auto-deployed environment, and load a run using `/api/runs/{id}`

### CODE REVIEW

**Comment by lukebjerring** [comment]
Staging instance deployed by Travis CI!
Running at https://test-run-id-dot-wptdashboard-staging.appspot.com

File: api/README.md [hunk 1]
```diff
@@ -7,21 +7,11 @@ document covers usage and parameters of those endpoints.
 
 An exhaustive list of the endpoints can be found in `routes.go`.
 
- - [/api/run](#apirun)
  - [/api/runs](#apiruns)
+ - [/api/runs/{id}](#apirun)
```
[the remark this answers is not in the document]
Comment (lukebjerring) [reply api/README.md]: Done.

File: api/README.md [hunk 2]
```diff
@@ -7,21 +7,11 @@ document covers usage and parameters of those endpoints.
 
 An exhaustive list of the endpoints can be found in `routes.go`.
 
- - [/api/run](#apirun)
```
[the remark this answers is not in the document]
Comment (lukebjerring) [reply api/README.md]: Reverted.

File: api/test_run.go [hunk 3]
```diff
@@ -27,51 +29,62 @@ func apiTestRunHandler(w http.ResponseWriter, r *http.Request) {
...
+	var testRun shared.TestRun
+	if idParam != "" {
+		id, err := strconv.ParseInt(idParam, 10, 0)
+		if err != nil {
+			http.Error(w, fmt.Sprintf("Invalid id '%s'", idParam), http.StatusBadRequest)
+			return
+		}
+		key := datastore.NewKey(ctx, "TestRun", "", id, nil)
+		if err = datastore.Get(ctx, key, &testRun); err != nil {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
```
[the remark this answers is not in the document]
Comment (lukebjerring) [reply api/test_run.go]: Good catch; throws an `ErrNoSuchEntity` when you fetch by key, so added a unit test for that.

**Comment by Hexcles** [comment]
LGTM again and merging.

**Pull request merged**